pangram/pangram.py

```diff
@@ -1,2 +1,9 @@
 def is_pangram(sentence):
-    pass
+    
+    # ensure string is lowercase
+    sentence = sentence.lower()
+    
+    # remove some punctuation
+    for thing in ' ,.!?:;()':
+        sentence = sentence.replace(thing,'')
+    return set(sentence) == set("abcdefghijklmnopqrstuvwxyz")
```

Comment:
set(sentence) == set("abcdefghijklmnopqrstuvwxyz") is an equality, what if you make set("abcdefghijklmnopqrstuvwxyz") in set(sentence), so that we don't have to treat differently symbols?

Reply:
Great idea, I hadn't thought of this. Refactoring...